### PR TITLE
crab

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by GamingGeek** [comment]
🦀

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ class ExampleModule(kdp: KDP) : Module(kdp, "example") {
             handler {
                 val user: User = args["user"] ?: error("User not found")
                 send("You are referring to ${user.asTag}!")
+                send("🦀")
```
Comment (Cubxity) [inline README.md]: That would leave user unused, ideally you should escape those emojis

**Review by Cubxity — CHANGES_REQUESTED** [review]

**Review by Cubxity — CHANGES_REQUESTED** [review]
user is still unused

**Comment by ghost** [comment]
agghg

**Review by Cubxity — APPROVED** [review]

**Pull request merged**